releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,576 @@
+
+    Matching the `max-error` configuration, the `lightning_task_info` configuration item records the name of the database that reports a data saving error.
+
+    This feature does not cover all types of errors, for example, syntax errors are not applicable.
+
+    [User document](/tidb-lightning/tidb-lightning-error-resolution.md#type-error)
+
+### Data sharing and subscription
+
+- Support replicating 100K tables simultaneously
+
+    By optimizing the data processing flow, TiCDC reduces the resource consumption of processing incremental data for each table, which greatly improves the replication stability and efficiency when replicating data in large clusters. The result of an internal test shows that TiCDC can stably support replicating 100,000 tables simultaneously.
+
+    [User document]()
+
+### Deployment and maintenance
+
+- Enable new collation rules by default
+
+    TiDB has supported new collation rules since v4.0, which behave the same as MySQL in the case-insensitive, accent-insensitive, and padding rules. The new collation rules are controlled by the `new_collations_enabled_on_first_bootstrap` parameter, which was disabled by default. Since v6.0, TiDB enables new collation rules by default. Note that this configuration takes effect only for the TiDB clusters first initialized.
+
+    [User documentation](/tidb-configuration-file.md#new_collations_enabled_on_first_bootstrap)
+
+- ​​Accelerate leader balancing after restarting TiKV nodes
+
+    After a restart, TiKV nodes need to redistribute the unevenly scattered leaders for load balance. In large-scale clusters, leader balancing time is positively correlated with the number of Regions. For example, the leader balancing of 100K Regions can take 20-30 minutes, which is prone to performance issues and stability risks due to uneven load. TiDB 6.0 provides a parameter to control the balancing concurrency and adjusts the default value to 4 times of the original, which greatly shortens the leader rebalancing time and accelerates the business recovery after a restart of TiKV nodes.
+
+    [User documentation](/pd-control.md#scheduler-show--add--remove--pause--resume--config)
+
+- Support canceling the automatic update of statistics
+
+    Statistics are one of the most important basic data that affects SQL performance. To ensure the completeness and timeliness of statistics, TiDB updates statistics periodically. Since v6.0, you can cancel the automatic update of statistics manually, which avoids any SQL performance impact caused by resource contention of the automatic update task.
+
+    [user documentation](/statistics.md#automatic-update)
+
+## Improvements
```

Comment:
```suggestion
### PingCAP Clinic diagnostic service (Technical Preview version)

PingCAP Clinic is a diagnostic service for TiDB clusters. This service helps to troubleshoot cluster problems remotely and provides a quick check of cluster status locally. With PingCAP Clinic, you can ensure the stable operation of your TiDB cluster for its full life cycle, predict potential problems, reduce the probability of problems, troubleshoot cluster problems quickly, and fix cluster problems.

When your cluster has some problems, you can contact PingCAP technical support for remote assistance. In this situation, you can help improve the troubleshooting efficiency by collecting and uploading diagnostic data using PingCAP Clinic service.

[User documentation](/clinic-introduction.md)

## Improvements
```

Reply:
```suggestion
### PingCAP Clinic diagnostic service (Technical Preview version)

PingCAP Clinic is a diagnostic service for TiDB clusters. This service helps users troubleshoot cluster issues remotely and provides a quick check of cluster status locally. With PingCAP Clinic, you can ensure the stable operation of your TiDB cluster for its full life cycle, predict potential issues, reduce the probability of issues, quickly troubleshoot and fix cluster issues.

When contacting PingCAP technical support for remote assistance to troubleshoot cluster issues, you can use the PingCAP Clinic service to collect and upload diagnostic data, thereby improving the troubleshooting efficiency.

[User documentation](/clinic-introduction.md)

## Improvements
```

Reply:
> 

this part is moved to line 356, so I will remove this one.